src/main/kotlin/org/jitsi/nlj/util/StreamInformation.kt

```diff
@@ -47,6 +47,9 @@ interface ReadOnlyStreamInformationStore {
     val rtpPayloadTypes: Map<Byte, PayloadType>
     fun onRtpPayloadTypesChanged(handler: RtpPayloadTypesChangedHandler)
 
+    fun getLocalPrimarySsrc(secondarySsrc: Long): Long?
```

Comment:
i only added these two (and not `getLocalSecondarySsrc` and `getRemotePrimarySsrc`) as we don't currently have a use case for them.